app/js/reducers/initialState.js

```diff
@@ -5,6 +5,7 @@ export default {
     orders: [],
     labTests: [],
     isLoading: false,
+    fetched: false,
```

Comment:
What is the difference between `fetched` and `isLoading`, I assume if a data is still being fetched then it is still loading.

Reply:
yeah @jomadoye so the isLoading will track the status of fetching just the orders and the fetched will track the status of completion for the fetch calls for that particular view